source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow.php

```diff
@@ -6,12 +6,12 @@
  */
 
 ?>
-<!-- wp:group {"align":"full","backgroundColor":"blueberry-1"} -->
-<div class="wp-block-group alignfull has-blueberry-1-background-color has-background">
-	<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"97px","bottom":"97px"},"blockGap":"0px"}},"className":"wporg-pattern__heading-with-arrow"} -->
-	<div class="wp-block-group alignfull wporg-pattern__heading-with-arrow has-link-color" style="padding-top:97px;padding-bottom:97px">
-		<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"300","lineHeight":"1.1","letterSpacing":"-0.02em"}},"fontSize":"heading-cta","fontFamily":"inter"} -->
-		<h2 class="has-inter-font-family has-heading-cta-font-size" style="font-style:normal;font-weight:300;letter-spacing:-0.02em;line-height:1.1"><a href="https://wordpress.org/download/">Get started</a></h2>
+<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"var:preset|spacing|60","right":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60"}}},"backgroundColor":"blueberry-1","textColor":"white","layout":{"inherit":true}} -->
+<div class="wp-block-group alignfull has-white-color has-blueberry-1-background-color has-text-color has-background has-link-color" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)">
+	<!-- wp:group {"align":"wide"} -->
+	<div class="wp-block-group alignwide">
+		<!-- wp:heading {"align":"wide","className":"is-style-with-arrow","fontSize":"heading-1","fontFamily":"inter"} -->
+		<h2 class="alignwide is-style-with-arrow has-inter-font-family has-heading-1-font-size"><a href="https://wordpress.org/download/">Get started</a></h2>
```

Comment:
This should be the CTA font size

Reply:
It was a mistake I noticed after I made the PR, and haven't circled back yet to fix it